Kconfig.xtos-build

```diff
@@ -35,3 +35,9 @@ config OPTIMIZE_FOR_NONE
 	  It means -O0 for GCC or equivalent for other compilers.
 
 endchoice
+
+config BUILD_VM_ROM
```

Comment:
I think this is needed for Zephyr as it's a ROM stub that jumps to SOF bootloader code in qemu.  Unless Zephyr has this jump built in when building a qemu target ?

Reply:
@lgirdwood I was wondering the same, but this is not used in any of the Zephyr builds and the implementation is in xtensa specific XTOS code, so I moved this out. But let me check. @andyross and @lyakh -- is this used in in Zephyr (to boot in Qemu) via some other way I don't see.

Reply:
IIRC to run Zephyr-generated SOF in Qemu I was using VMROM, built separately by a native SOF build. But this was a few months ago so I might be wrong